Uncertainty_Quantification/Deep500_recipe/losses.py

```diff
@@ -0,0 +1,36 @@
+import torch
+import torch.nn.functional as F
+from pytorch_msssim import MS_SSIM
+
+
+class MSSSIMLoss(torch.nn.Module):
+    def __init__(self, dim=2):
+        super(MSSSIMLoss, self).__init__()
+        if dim == 2:
+            self.msssim = MS_SSIM(data_range=4.3, channel=1)
+        else:
+            self.msssim = MS_SSIM(
+                data_range=4.3,
+                channel=2  # 60 for mean
+            )  # after standardization ~0+-60 through analysis, channel = 2 for 2 pressure levels
```

Comment:
2 for 2 pressure levels, what does that mean?

Reply:
This is just a copy from this comment:

https://github.com/spcl/deep-weather/blob/master/Uncertainty_Quantification/Pytorch/models.py#L28